integration/tests/basic/test_wneon.py

```diff
@@ -95,6 +97,7 @@ def test_withdraw(self, wneon):
         )
         assert wneon_balance_after == wneon_balance_before - withdraw_amount
 
+    @pytest.mark.mainnet
     def test_transfer_and_check_token_does_not_use_spl(self, wneon):
         deposit_amount = 10
```

Comment:
don't need, because we checked this contract in devnet and contract can't be changed

Reply:
removed